boards/nordic/nrf5340_audio_dk/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -191,7 +203,7 @@ arduino_serial: &uart1 {
 		spi-max-frequency = <8000000>;
 	};
 
-	cs47l63: cs47l63@1 {
+	cs47l63: cs47l63@2 {
```

Comment:
Why was this changed to 2?

Reply:
It was suggested by anangl above that shield users will probably use CS0, hence sdhc and cs47l63 should be incremented by 1.